templates/quickstart-jira-dc.template.yaml

```diff
@@ -1035,114 +1006,51 @@ Resources:
 
-                localansiblerepopath=/home/localansible
-                gitkeylocation=/root/.ssh/gitkey
+            /opt/atlassian/bin/clone_deployment_repo:
+              content: !Sub |
+                #!/bin/bash
+                key_location=/root/.ssh/deployment_repo_key
+                key_name="${DeploymentAutomationKeyName}"
 
-                # ensure awscli is up to date
-                pip install --upgrade awscli
-                gitkey=$(aws --region=${AWS::Region} ssm get-parameters --names "${LocalAnsibleGitSshKeyName}" --with-decryption | jq --raw-output '.Parameters[0] .Value')
-                echo -e $gitkey > $gitkeylocation
-                chmod 600 $gitkeylocation
-                export GIT_SSH_COMMAND="ssh -o IdentitiesOnly=yes -o StrictHostKeyChecking=no -i $gitkeylocation"
+                yum install -y git
+                if [[ ! -z "$key_name" ]]; then
+                    # Ensure awscli is up to date
+                    yum install -y awscli jq
+                    key_val=$(aws --region=${AWS::Region} ssm get-parameters --names "$key_name" --with-decryption | jq --raw-output '.Parameters[0] .Value')
+                    echo -e $key_val > $key_location
+                    chmod 600 $key_location
+                    export GIT_SSH_COMMAND="ssh -o IdentitiesOnly=yes -o StrictHostKeyChecking=no -i $key_location"
+                else
+                    export GIT_SSH_COMMAND="ssh -o IdentitiesOnly=yes -o StrictHostKeyChecking=no"
+                fi
 
-                mkdir -p $localansiblerepopath
-                cd $localansiblerepopath
-                git clone "${LocalAnsibleGitRepo}"
-                cp $localansiblerepopath/*/local-ansible-run /usr/local/bin/local-ansible-run
-                chmod 750 /usr/local/bin/local-ansible-run
+                git clone "${DeploymentAutomationRepository}" -b "${DeploymentAutomationBranch}" /opt/atlassian/dc-deployments-automation/
               mode: "000750"
               owner: root
               group: root
+
           commands:
-            010_make_mount_point:
-              command: mkdir -p /media/atl
-              ignoreErrors: false
-            020_add_nfs_mount:
-              command:
-                !Join
-                  - ""
-                  -
-                    - "echo "
-                    - !If
-                      - UseHostedZone
-                      - !Ref EFSCname
-                      - !Sub
-                          - $(curl -s http://169.254.169.254/latest/meta-data/placement/availability-zone).${ElasticFileSystem}".efs."${Region}".amazonaws.com"
-                          - ElasticFileSystem: !Ref ElasticFileSystem
-                            Region: !Ref "AWS::Region"
-                    - ":/ /media/atl nfs4 rw,vers=4.1,rsize=1048576,wsize=1048576,hard,timeo=600,retrans=2 0 0 >>/etc/fstab"
+            070_create_atl_dir:
```

Comment:
Should we renumber those commands and start from 010 again?

Reply:
I usually space them by priority, e.g. the ansible run is a high number as it should be near the end.